app/src/main/java/layout/mapFragment.java

```diff
@@ -84,8 +90,11 @@
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View rootView = inflater.inflate(R.layout.fragment_map, container, false);
 
+        FeelTripApplication.setFrag(frag); //TODO: Put this wherever we initialize the fragment, I think here works fine.
+
```

Comment:
This may change. If it does, then we'll have to place this code segment accordingly. As of right now it works though